### PR TITLE
Add icons and placeholder illustrations

### PR DESCRIPTION
**Pull request by kanchanagodage**
## Purpose
In order to improve user experience designed 68 new icons and 3 placeholders illustrations. 15 icons were replaced.

## Icon Pack
Icon Pack@2x

## Replaced Icons
icons-replace@2x

## Placeholder Illustrations
Placeholders@2x

### CODE REVIEW

File: modules/theme/src/themes/dark/assets/images/icons/building-icon.svg [hunk 1]
```diff
@@ -1,5 +1,3 @@
-<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (DimalChandrasiri) [inline modules/theme/src/themes/dark/assets/images/icons/building-icon.svg:1]: Could you explain the reason to remove this line?

File: modules/theme/src/themes/dark/assets/images/icons/code-fork.svg [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!--
- * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (DimalChandrasiri) [inline modules/theme/src/themes/dark/assets/images/icons/code-fork.svg:2]: Unless the file is removed, AFAIK the initial year should not be changed.

Comment (kanchanagodage) [reply modules/theme/src/themes/dark/assets/images/icons/code-fork.svg:2]: The previous icon was removed. It replaced with a new icon.

File: modules/theme/src/themes/dark/assets/images/icons/plus-icon.svg [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <!--
- * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (DimalChandrasiri) [inline modules/theme/src/themes/dark/assets/images/icons/plus-icon.svg:2]: Should be `2019` since the intial file is added in 2019

Comment (kanchanagodage) [reply modules/theme/src/themes/dark/assets/images/icons/plus-icon.svg:2]: The previous icon replaced with a new icon.

**Review by DimalChandrasiri — APPROVED** [review]

File: modules/theme/src/themes/dark/assets/images/icons/plus-icon.svg (hunk 3 above)
Comment (kanchanagodage) [reply modules/theme/src/themes/dark/assets/images/icons/plus-icon.svg]: Fixed

File: modules/theme/src/themes/dark/assets/images/icons/code-fork.svg (hunk 2 above)
Comment (kanchanagodage) [reply modules/theme/src/themes/dark/assets/images/icons/code-fork.svg]: Fixed

File: modules/theme/src/themes/dark/assets/images/icons/building-icon.svg (hunk 1 above)
Comment (kanchanagodage) [reply modules/theme/src/themes/dark/assets/images/icons/building-icon.svg]: Fixed

**Pull request merged**